### PR TITLE
bump(deps): protobuf, grpc for open-telemetry

### PR DESCRIPTION
**Pull request by sallyom**
Signed-off-by: Sally O'Malley <somalley@redhat.com>

#### What type of PR is this?

/kind dependency-change

#### What this PR does / why we need it:
protobuf and grpc update
running through CI to detect issues

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -69,13 +69,13 @@ require (
...
 
 replace (
-	github.com/golang/protobuf => github.com/golang/protobuf v1.3.5
+	github.com/golang/protobuf => github.com/golang/protobuf v1.4.3
 	github.com/opencontainers/runtime-spec => github.com/opencontainers/runtime-spec v1.0.3-0.20201121164853-7413a7f753e1
 	// Pinning the syndtr/gocapability until https://github.com/opencontainers/runc/commit/6dfbe9b80707b1ca188255e8def15263348e0f9a
 	// is included in the runc release
 	github.com/syndtr/gocapability => github.com/syndtr/gocapability v0.0.0-20180916011248-d98352740cb2
 	google.golang.org/genproto => google.golang.org/genproto v0.0.0-20200117163144-32f20d992d24
-	google.golang.org/grpc => google.golang.org/grpc v1.27.0
```
Comment (saschagrunert) [inline go.mod:78]: There was a kata containers related reason to pin the dependency here (https://github.com/cri-o/cri-o/pull/4164). Is this obsolete now? 

/cc @fidencio 

If so, then I would recomment removing the lines completely to automatically vendor `google.golang.org/grpc v1.37.0`.

Comment (sallyom) [reply go.mod:78]: I'll ask around, I dunno if this is still necessary.

**Comment by sallyom** [comment]
I'll close this as I've included the bump in the tracing PR: https://github.com/cri-o/cri-o/pull/4883. I've moved the comment about whether kata needs protobuf pinned there, also.

**Pull request closed without merging**